packages/dashboard/src/components/doors-app.tsx

```diff
@@ -43,36 +120,31 @@ export const DoorsApp = createMicroApp('Doors', () => {
   }, [rmf]);
 
   return (
-    <Grid container>
-      {buildingMap &&
-        buildingMap.levels.flatMap((level) =>
-          level.doors.map((door) => (
-            <DoorCard
-              key={door.name}
-              name={door.name}
-              level={level.name}
-              type={door.door_type}
-              sx={{ width: 200 }}
-              aria-labelledby={`door-cell-${door.name}`}
-            >
-              <CardActions sx={{ justifyContent: 'center' }}>
-                <DoorControls
-                  doorName={door.name}
-                  onOpenClick={() =>
-                    rmf?.doorsApi.postDoorRequestDoorsDoorNameRequestPost(door.name, {
-                      mode: RmfDoorMode.MODE_OPEN,
-                    })
-                  }
-                  onCloseClick={() =>
-                    rmf?.doorsApi.postDoorRequestDoorsDoorNameRequestPost(door.name, {
-                      mode: RmfDoorMode.MODE_CLOSED,
-                    })
-                  }
-                />
-              </CardActions>
-            </DoorCard>
-          )),
-        )}
-    </Grid>
+    <Table stickyHeader size="small" aria-label="door-table" style={{ tableLayout: 'fixed' }}>
+      <TableHead>
+        <TableRow>
+          <TableCell>Name</TableCell>
+          <TableCell>Level</TableCell>
+          <TableCell>Type</TableCell>
+          <TableCell>Door State</TableCell>
+          <TableCell></TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {buildingMap &&
+          buildingMap.levels.flatMap((level) =>
+            level.doors.map((door) => (
+              <DoorCard
```

Comment:
Similar to my comment on lift cards, we will stop using `DoorCard` and we should move to something new implemented here in `doors-app`, like `DoorTableRow` and `DoorTableProp`, and we should be able to remove that dependency now.